servers/alhena/default.nix

```diff
@@ -0,0 +1,20 @@
+{ modulesPath, inputs, config, pkgs, ... }:
+let
+  vs = config.vault-secrets.secrets;
+in {
+  imports = [
+    "${modulesPath}/virtualisation/amazon-image.nix"
+    inputs.hermetic.nixosModules.hermetic
+  ];
+
+  systemd.services.amazon-init.enable = false;
+
+  vault-secrets.secrets.hermetic.environmentPrefix = "hermetic";
+  services.hermetic = {
+    enable = true;
+    environmentFile = "${vs.hermetic}/environment";
+    package = inputs.hermetic.defaultPackage.${pkgs.stdenv.system};
```

Comment:
Why isn't this set in the hermetic repo by default?

Reply:
Why is what not set? Feel free to add an overlay to the hermetic flake, then I can just remove this line since the module defaults to `pkgs.hermetic`.

Reply:
I'm not sure if that would be much better, it would be nice to simply have the module's default point directly to the package, but platforms/systems make that impossible (nixosModules is not per-platform)

Reply:
No, but an overlay would solve it. It's out of scope for this PR though, feel free to tackle separately.